src/core/reference/src/utils/coordinate_transform.cpp

```diff
@@ -11,17 +11,16 @@
 #include <sstream>
 #include <vector>
 
-#include "ngraph/axis_vector.hpp"
-#include "ngraph/coordinate_diff.hpp"
-#include "ngraph/except.hpp"
-#include "ngraph/shape.hpp"
-#include "ngraph/strides.hpp"
 #include "ngraph/util.hpp"
```

Comment:
Do we still need to have `ngraph/util.hpp`?

Reply:
Yes, ngraph::ceil_div and ngraph::subtract_or_zero are called in a depracated part.